bindings/nodejs/src/lib.rs

```diff
@@ -293,16 +327,15 @@ impl Operator {
     /// ```
     #[napi]
     pub async fn append(&self, path: String, content: Either<Buffer, String>) -> Result<()> {
-        let c = match content {
-            Either::A(buf) => buf.as_ref().to_owned(),
-            Either::B(s) => s.into_bytes(),
-        };
-
-        self.0
-            .write_with(&path, c)
-            .append(true)
-            .await
-            .map_err(format_napi_error)
+        self.write(
```

Comment:
Tracked at https://github.com/apache/opendal/issues/4792

Maybe you want to implement this too?

Reply:
Yes, I can do it, plz assign it to me